src/models/VotesInfo.ts

```diff
@@ -0,0 +1,55 @@
+import { TOCCode } from '../data/TocData'
+
+export default class VotesInfo {
+  /**
+   * A valid TOC reporting mark.
+   *
+   * @memberof VotesInfo
+   */
+  tocReportingMark: TOCCode
+
+  /**
+   * Number of votes for this TOC. `null` if poll hasn't started.
+   *
+   * @memberof VotesInfo
+   */
+  votes: number | null
+
+  /**
+   * Vote history over time. Usually taken every few mins, but depends on Twitter API's update rate. `null` if poll hasn't started.
+   *
+   * @memberof VotesInfo
+   */
+  votingHistory: VotingHistoryEntry[] | null
+
+  /**
+   * Creates an instance of VotesInfo.
+   *
+   * @memberof VotesInfo
+   */
```

Comment:
```suggestion
  /**
   * A valid TOC reporting mark.
   *
   * @memberof VotesInfo
   */
  public tocReportingMark: TOCCode

  /**
   * Number of votes for this TOC. `null` if poll hasn't started.
   *
   * @memberof VotesInfo
   */
  public votes: number | null

  /**
   * Vote history over time. Usually taken every few mins, but depends on Twitter API's update rate. `null` if poll hasn't started.
   *
   * @memberof VotesInfo
   */
  public votingHistory: VotingHistoryEntry[] | null

  /**
   * Creates an instance of VotesInfo.
   *
   * @memberof VotesInfo
   */
```

Reply:
I'd say we shouldn't add `public` to the fields as Typescript is public by default.

https://www.typescriptlang.org/docs/handbook/classes.html#public-by-default

Reply:
So previously I've had a linter say, "you should mark these" (even though it is public by default), so I just assumed it was best practice.  However, a quick search shows: basically, it boils down to personal preference.

So, my personal preference is to have the access modifiers. Whether or not to add them is down to you.